.github/workflows/test.yml

```diff
@@ -15,13 +15,13 @@ jobs:
       fail-fast: false
       matrix:
         sage-version:
-          - '9.2'
-          - '9.3'
           - '9.4'
           - '9.5'
           - '9.6'
           - '9.7'
           - '9.8'
+          - '10.1'
+          - '10.2'
```

Comment:
Worth testing 10.0 too?

Reply:
I think there is no docker for sage 10.0. But maybe you use conda instead ?

Reply:
Yeah currently the workflow uses conda